components/datepicker/modules/calendar.vue

```diff
@@ -81,17 +88,79 @@ export default {
 
   watch: {
     calendarDays () {
-      // local highlightDay is reset when calendarDays is updated
-      this.highlightDay = null;
+      // on calendarDays update, reset our local variables
+      this.focusDay = 0;
+      this.daysRef = [];
+      this.selectedDay = null;
     },
   },
 
   methods: {
+    setDayRef (el, currentMonth) {
+      if (!this.daysRef.includes(el) && currentMonth) {
+        this.daysRef.push(el);
+      }
+    },
+
+    handleKeyDown (event) {
+      switch (event.key) {
+        case 'ArrowUp':
+          event.preventDefault();
+          this.focusDay -= 7;
+          try {
+            this.daysRef[this.focusDay].focus();
+          } catch (error) {
+            this.$emit('focus-month-year-picker');
+          }
+          break;
+
+        case 'ArrowDown':
+          event.preventDefault();
+          this.focusDay += 7;
+          try {
+            this.daysRef[this.focusDay].focus();
+          } catch (error) {
+            this.$emit('focus-month-year-picker');
+          }
+          break;
```

Comment:
I don't like this behavior of changing to `month-year-picker` on `arrow-up/down`, I'll prefer to change to next, previous month with `arrow-left/right`.

Here's an example https://material.angular.io/components/datepicker/overview

Reply:
It is changing next/previous month with arrow-left/right, arrow-up/down is just an addition.
We have it in the emoji picker, you can move trought the tabset and search with arrow-up/down.

I don't see the problem to have this additional navigation. I see it useful.

Reply:
This is one of those opinionated accessibility things that could be argued for or against. I'm okay leaving it the way it is now unless we have customers / product developers complaining about it.